README.md

```diff
@@ -23,6 +23,7 @@ __Windows__
 * [Visual C++ Redistributable Packages for Visual Studio 2015](http://www.microsoft.com/en-us/download/details.aspx?id=48145)
 * [Cmake 3.1.0+](http://www.cmake.org/download/) (optional, required only for LLVM build; add to PATH)
 * [Python](https://www.python.org/downloads/) (optional, required only for LLVM build; add to PATH)
+* [Git] (https://git-scm.com/downloads) (Select "use Git from the Windows Command Prompt")
```

Comment:
@tambry But it is written in the README:
Building

To initialize the repository don't forget to execute git submodule update --init to pull the wxWidgets source.
So what you told me it is totally contradictory to the.
The Readme told launched a git, for not say against it downloaded, it's completely illogical.
Should perhaps be doing something in the order.
And no, it's links has nothing to do with Windows, it is for all the OS is Bigpet who had done me changed.